addons/Media_Recorder/src_webpack/MediaRecorder.jsm

```diff
@@ -216,7 +225,20 @@ export class MediaRecorder {
     reset() {
         this.deactivate();
         if (this.model.isResetRemovesRecording) {
-            this.resetRecording();
+            if (this.isRecording) {
+                this.resetRecording();
+            } else {
+                this.resetRecording();
```

Comment:
Mam wrażenie, że jest to pozostałość po starym kodzie. Jeżeli nie jest to pozostałość prosiłbym o poprawę (wyciagnięcie wspólnej części przed if). Proszę o potwierdzenie, że nie jest to pozostałość.
Jest to poprawka nieblokująca o ile nie jest to pozostałość.